atomic_scanners/container-capabilities-scanner/scanner.py

```diff
@@ -80,10 +80,12 @@ def template_json_data():
         "Scan Results": {"Container capabilities": None},
         # "Docker run command": "docker run -d {} tailf /dev/null".format(
         #     IMAGE_NAME),
-        "Reference documentation": "http://www.projectatomic.io/blog/2016/01/how-to-run-a-more-secure-non-root-user-container/"
+        "Reference documentation": "http://www.projectatomic.io/blog/2016/01/how-to-run-a-more-secure-non-root-user-container/",
+        "Summary": None
```

Comment:
Summary key in dict will hold text value, as a point of having uniform values a key can get, a default value equivalent to `None` could be `""`.